Contributing.md

```diff
@@ -0,0 +1,20 @@
+# Contributing:
```

Comment:
You can change the title to `Contributing Guidelines`

Reply:
@luckyman20  done!